public/sketch.js

```diff
@@ -14,7 +14,7 @@ function setup() {
 }
 
 function draw() {
-  image(video, 0, 0, width, height);
+  image(video.get(), 0, 0, width, height);
```

Comment:
```suggestion
  // Issue related to: https://github.com/processing/p5.js/issues/4535#issuecomment-640072884
  image(video.get(), 0, 0, width, height);
```

Reply:
Wow how do you suggest changes like this? This is awesome!